manifests/t/Teoi/Lua/5.4.4/Teoi.Lua.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Teoi.Lua
+PackageVersion: 5.4.4
+PackageLocale: en-US
+Publisher: Teoi

```

This should match what is written to control panel
```suggestion
Publisher: Humanity
```